src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: LAMPy stack monitoring integration
+tags:
+    - New Relic integrations
+    - LAMPy
+    - Linux
+    - Apache
+    - Python
+metaDescription: Use New Relics Linux, Apache, Python and MySql monitoring to get a dashboard with metrics from your LAMPy application.
+
+---
+
+import infrastructureLampyDashboard from 'images/infrastructure_screenshot-full_lampy-dashboard.png'
+
+Our LAMPy integration makes use of the Infrastructure agent, MySQL agent, Apache agent, and Python agent to provide you with a pre-built dashboard with your most important metrics like response time, CPU utilization, traffic, and login frequencies.
```

Comment:
I think in a lot of these docs we're calling many things 'agents' that we shouldn't be. We don't call our MySQL and Apache integrations 'agents' we just call them integrations. We reserve 'agent' for a few things, like the APM language agents, the infrastructure agent, browser agent, mobile agents. so I think we should reword this entire doc (and probably some docs we've already put out) to differentiate between agents and integrations. in this case, 'MySQL integration, Apache integration...'